### ISSUE TITLE
Multiterm shell popup won't open

### ISSUE THREAD

**Issue by nwolfe** [syl20bnr/spacemacs#5719]
#### Description
The multi-term shell fails to popup due to incorrect number of args passed to function.

#### Reproduction guide
- Start Emacs
- SPC a s m

*Observed behaviour:*
Error message at the bottom saying:
> Wrong number of arguments: (lambda (_) "Wrapper to be able to call multi-term from shell-pop" (interactive) (multi-term)), 0

*Expected behaviour:*
Popup window with multi-term

#### System Info
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.16
- Spacemacs branch: develop (rev. 2b4a039)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-enable-help-tooltip t auto-completion-enable-snippets-in-popup t)
 better-defaults
 (clojure :variables clojure-enable-fancify-symbols t)
 (colors :variables colors-enable-rainbow-identifiers t)
 deft emacs-lisp git github markdown
 (mu4e :variables mu4e-installation-path "/usr/local/share/emacs/site-lisp")
 org osx puppet python ruby
 (shell :variables shell-default-shell 'multi-term shell-default-height 40 shell-default-term-shell "/bin/zsh")
 spacemacs-layouts spotify
 (spell-checking :variables spell-checking-enable-by-default nil)
 version-control yaml)
```

#### Backtrace
```
Debugger entered--Lisp error: (wrong-number-of-arguments (lambda (_) "Wrapper to be able to call multi-term from shell-pop" (interactive) (multi-term)) 0)
  multiterm()
  call-interactively(multiterm nil)
  (lambda nil (call-interactively (quote multiterm) nil))()
  shell-pop--switch-to-shell-buffer(1)
  shell-pop-up(1)
  shell-pop(nil)
  (if (equal (quote (4)) index) (call-interactively (quote multiterm)) (shell-pop--set-shell-type (quote shell-pop-shell-type) (quote ("multiterm" "*multiterm*" (lambda nil (call-interactively (quote multiterm) nil))))) (shell-pop index))
  spacemacs/shell-pop-multiterm(nil)
  call-interactively(spacemacs/shell-pop-multiterm nil nil)
  command-execute(spacemacs/shell-pop-multiterm)
```

**Comment by nwolfe** [comment]
This is a result of the recent https://github.com/syl20bnr/spacemacs/commit/63dc8f7ea0b262e3bd1e164bda87d7e1265adfcf change.

I believe I have a fix and will submit a PR.

**Comment by xged** [comment]
Still not working in v0.105.4-855-g5c7e0b6. 
Error message: 
> Wrong number of arguments: (lambda nil "Wrapper to be able to call multi-term from shell-pop" (interactive) (multi-term)), 1

**Comment by nwolfe** [comment]
Yep, the cause for this bug was reverted 😄 

Fixed here https://github.com/syl20bnr/spacemacs/pull/5743

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
